test/Spec.hs

```diff
@@ -24,6 +25,7 @@ hspecTests = sequential . joinSpecs "issue-wanted" []
 hedgehogTests :: AppEnv -> Group
 hedgehogTests env = Group "Roundtrip properties" 
     [ issueRoundtripProp env `named` "fromRow . toRow ≡ id"
+    , repoRoundtripProp env `named` "fromRow . toRow ≡ id" 
```

Comment:
In terminal you will this text. So let's add some context to it to understand which test this is for:

```haskell
"Issue: fromRow . toRow ≡ id"
"Repo: fromRow . toRow ≡ id"
```